samples/bluetooth/mesh/light_ctrl/src/lc_pwm_led.c

```diff
@@ -49,9 +33,5 @@ void lc_pwm_led_set(uint16_t desired_lvl)
 		(PWM_PERIOD * desired_lvl) /
```

Comment:
Should we use `PWM_PERIOD` here or rather take it from the DTS spec?

Reply:
I replaced the call below with `pwm_set_dt` and used `PWM_PERIOD` there.